README.md

```diff
@@ -11,6 +11,9 @@ duplicated git dependencies.
 
 ## Local chain usage
 
+First make sure you have [`compactc`](https://docs.midnight.network/relnotes/compact) set up and the `COMPACT_HOME` env var set to the `compactc` directory.
+You must set the environmental vars `ZKIR_PP` and `MIDNIGHT_LEDGER_STATIC_DIR` to the `local-chain-setup/midnight-static-dir/` directory in this repo.
```

Comment:
maybe I should move those out of `local-chain-setup` considering that currently those are the same for the testnet anyway?

Reply:
It's hand to have them here though. That way you can build it without having to obtain those files yourself.